python/example/single_cell_allen.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+
+from collections import defaultdict
+import json
+import arbor
+import seaborn
+import pandas
+import matplotlib.pyplot as plt
+
+# (3) A function that parses the Allen parameter fit file into components for an arbor.decor
+# NB. Needs to be adjusted when using a different model
+def load_allen_fit(fit):
+    with open(fit) as fd:
+        fit = json.load(fd)
+
+    # cable parameters convenience class
+    class parameters:
```

Comment:
This is not idiomatic Python. You are setting members to a _class_ not to an _instance_?
```
In [19]: class A:
    ...:     a = 42
    ...:
In [22]: o = A()
In [23]: o.a
Out[23]: 42
In [24]: p = A()
In [25]: p.a
Out[25]: 42
In [26]: A.a = 23
In [27]: p.a
Out[27]: 23
In [28]: o.a
Out[28]: 23
```

It works here though, for reasons known only to Python... Still I wouldn't want to advertise this
as proper. Either use `dataclass` like my original code does or namedtuple, see 
https://docs.python.org/3/library/collections.html#namedtuple-factory-function-for-tuples-with-named-fields
and examples therein.

Reply:
```
    class parameters:
        def __init__():
            self.cm = None
            self.tempK = None
            self.Vm = None
            self.rL = None
```

Reply:
mystery solved, we **add** instance variables _over_ class variables, which weird, but works.

Reply:
See new commit!

```
    class parameters:
        def __init__(self, **kwargs):
            self.cm = None
            self.tempK = None
            self.Vm = None
            self.rL = None
            self.__dict__.update(kwargs)
```